app/assets/javascripts/components/items_list.jsx

```diff
@@ -0,0 +1,15 @@
+class ItemsList extends React.Component {
+  render() {
+    const items = this.props.items
+	const user = this.props.user;
+	return(
+	  <select className="form-control selcls" value={this.props.value} onChange={this.props.change} >
+		{
```

Comment:
please move this generation in separate function.

Reply:
Will be fixed.